### ISSUE TITLE
Combine and remove duplicates

### ISSUE THREAD

**Issue by lonelykid** [DIYgod/RSSHub#6636]
### 这是一个什么样的功能？
A special route with other routers as args. Combine the entries and remove duplicates. 

### 这个功能可以解决什么问题？
Subscribe to a set of feeds with one xml without duplicates. For example ``/caixin/article`` and ``/caixin/latest`` use different apis. Some contents are the same but some are not. Suppose you want to get as many articles as possible. You have to subscribe to both and get many duplicates. I assume this is not uncommon for media websites.

**Comment by lonelykid** [comment]
https://github.com/DIYgod/RSSHub/issues/6627 @thedogb
This is related. A special route to process existing xml would be helpful.

**Comment by NeverBehave** [comment]
对于内容的处理需要更多讨论

**Comment by lonelykid** [comment]
This is what I have in mind.
## On multiple feeds
 1. Combine. RSSHub internal and external feeds.
 2. Remove duplicates.

``rsshub.app/combine/encodeURIComponent(urls.join(';'))?remove_duplicate=1``
urls are RSSHub route. No need to support external xml. Another route can be created to convert it.

## On single feed
### External
``rsshub.app/rproxy/url``
full url to an external xml. convert xml to rsshub route.

### Conversion
Some are already available. https://docs.rsshub.app/parameter.html

 3. ✅ Filter. keywords. 
 4. ✅ Simple text conversion. Traditional/Simplified Chinese. 
 5. Extract full text. nice to have, but already declined https://github.com/DIYgod/RSSHub/issues/6003

There does not seem anything to do for now.

Basically, I think two routes can be added at least.

**Comment by NeverBehave** [comment]
- Combining feeds may be done on local: people may combine two unrelated feeds which heavier our work. I do want to have a timeline like RSS feed experience and it maybe done on other project.
- RSS proxy, the same reason, RSShub has lots of scripts to take care, these work could be separated and done on a different project. 

Adding extra layer or middlewares make it too complex. We could possibly do this via e.g. `rssproxy.app` that takes multiple rsshub links and handle these conversions. RSShub shouldn't be like an general API endpoint for data cleaning or filtering IMO.

**Comment by lonelykid** [comment]
OK. I understand. Indeed these can be done via a separate project if your envision is keeping RSSHub simple. But this might not add much maintenance burden. It does not need to deal with constantly changing web interface.

I know some RSS reader can do that. I use Inoreader. It's a premium feature. The basic paid plan does not support it :(

**Comment by NeverBehave** [comment]
https://github.com/DIYgod/RSSHub/issues/122

**Comment by NeverBehave** [comment]
I would say it is a great idea to start a small, but experimental project and test how it works. This idea has been asked since the project begins.

**Comment by NeverBehave** [comment]
> OK. I understand. Indeed these can be done via a separate project if your envision is keeping RSSHub simple. But this might not add much maintenance burden. It does not need to deal with constantly changing web interface.
> 
> I know some RSS reader can do that. I use Inoreader. It's a premium feature. The basic paid plan does not support it :(

Actually if we have time, we are about to switch to improve overall quality: scripts are breaking all the time and we don't even have a reliable way to identify them. There are lots of stuff to take care for sure, and moving this part away would ease the pain.
